src/scripts/templates/gift_card.js

```diff
@@ -1,32 +1,35 @@
+import QRCode from 'qrcode';
+
 /**
  * Gift Card Template Script
  * ------------------------------------------------------------------------------
  * A file that contains scripts highly couple code to the Gift Card template.
  */
 
-import $ from 'jquery';
-
-const config = {
-  qrCode: '#QrCode',
-  printButton: '#PrintGiftCard',
-  giftCardCode: '.giftcard__code',
+const selectors = {
+  qrCodeCanvas: '[data-gift-card-qr]',
+  printButton: '[data-gift-card-print]',
+  giftCardCode: '[data-gift-card-digits]',
 };
 
-// new QRCode($qrCode[0], {
-//   text: $qrCode.attr('data-identifier'),
-//   width: 120,
-//   height: 120,
-// });
+// This is the QR code that allows customers to use at a POS
+document.querySelectorAll(selectors.qrCodeCanvas).forEach((qrCodeCanvas) => {
+  QRCode.toCanvas(qrCodeCanvas, qrCodeCanvas.getAttribute('data-identifier'));
```

Comment:
Alternatively you can use `.dataset` to directly access `data-` values.
```suggestion
  QRCode.toCanvas(element, element.dataset.identifier);
```